critiquebrainz/frontend/templates/macros.html

```diff
@@ -51,3 +51,11 @@
     </script>
   {% endif %}
 {% endmacro %}
+
+{% macro review_credit(review) %}
+{{ _(
+  'Event review by %(user)s' if review.entity_type == 'event' else 'Music review by %(user)s' if review.entity_type == 'release_group' else 'Review by %(user)s',
```

Comment:
`Music` is way too general. It's probably better to just put `Release group` there.